### ISSUE TITLE
When use `indentedSyntax` option, have to use `.scss` to import scss file

### ISSUE THREAD

**Issue by indigofeather** [jtangelder/sass-loader#135]
Hi, I got a problem, when used `indentedSyntax` option, it occurred error:

```
ERROR in ./~/css-loader?sourceMap!./~/sass-loader?indentedSyntax&sourceMap&sourceMapContents!./src/css/application.sass
    Module build failed: 
    @import 'foundation/reset'
           ^
          File to import not found or unreadable: foundation/reset.sass
          in /home/**/src/css/application.sass (line 12, column 9)
```

Before using `sass-loader`, for example, we have `foundation/_reset.scss` file, in normal usage (Ex: ruby-sass, node-sass):
```sass
// -------------------------------------
//   Foundation
// -------------------------------------

@import 'foundation/reset'
@import 'foundation/helpers'
@import 'foundation/config'
@import 'foundation/base'
```
use sass-loader, we have to be:
```sass
// -------------------------------------
//   Foundation
// -------------------------------------

@import 'foundation/reset.scss'
@import 'foundation/helpers'
@import 'foundation/config'
@import 'foundation/base'
```
---
But it's not necessarily to set extension for these files.

in `index.js` @ line 140:
```js
// indentedSyntax is a boolean flag
opt.indentedSyntax = Boolean(opt.indentedSyntax);
fileExt = '.' + (opt.indentedSyntax? 'sass' : 'scss');
```

in `index.js` @ line 92:
```js
if (!url.match(extensionMatcher)) {
    extension = contextMatch && contextMatch[0] || fileExt;
    url = url + extension;
}
```

Without above, it works fine!
Any insight would be appreciated!

**Comment by Fraina** [comment]
+1, it would be awesome if I don't have to check extension before import module's stylesheets.

e.g. 
```sass
@import './node_modules/susy/sass/susy'
@import './node_modules/breakpoint-sass/stylesheets/breakpoint'
@import 'foundation/susy_config'
```
instead of
```sass
@import './node_modules/susy/sass/susy.scss'
@import './node_modules/breakpoint-sass/stylesheets/breakpoint.scss'
@import 'foundation/susy_config'
```

**Comment by jorrit** [comment]
I am assuming you have `scss` and `sass` in your `resolve.extensions` of the webpack.config.js file?

**Comment by indigofeather** [comment]
Hello, @jorrit 
I just use `['', '.js']` in `resolve.extensions`, and `loaders` has:

```js
  {
    test: /\.scss$|\.sass$/,
    loader: ExtractTextPlugin.extract('css-loader?sourceMap!autoprefixer-loader?{browsers:["last 2 version"]}!sass-loader?indentedSyntax&sourceMap&sourceMapContents')
  }
```

**Comment by jhnns** [comment]
Could you guys check out the current master branch and check if it works for you? I've heavily refactored the import mechanism and need more tests.

**Comment by indigofeather** [comment]
Hello @jhnns 
I've tested the current master branch, it works!

**Comment by jhnns** [comment]
Thx. That's what I wanted to hear :+1: :grinning:.

**Comment by indigofeather** [comment]
Waiting for the next version :+1:

**Comment by jhnns** [comment]
Shipped with 2.0.0
